frontend/src/components/editor/links/cell-link.tsx

```diff
@@ -4,22 +4,27 @@ import { Logger } from "../../../utils/Logger";
 import { cn } from "@/utils/cn";
 import { displayCellName } from "@/core/cells/names";
 import { useCellActions, useCellIds, useCellNames } from "@/core/cells/cells";
+import { useFilename } from "@/core/saving/filename";
```

Comment:
this might be the reason for the failing tests. 

in `src/core/saving/filename.ts:6:42` we could add `if (import.meta.vitest)` and not check the dom

Reply:
done! import.meta.env.MODE == "test" seemed the most reliable
but the assert also seems a bit strong in this context?

Reply:
Yes that is better - there is no reliable fallback. I'd rather fail hard so it's obvious than silently and let things sneak through